packages/mdc-menu/index.js

```diff
@@ -111,85 +130,100 @@ class MDCMenu extends MDCComponent {
     }
   }
 
-  /** @param {number} index */
-  set selectedItemIndex(index) {
-    this.foundation_.setSelectedIndex(index);
+  /** @param {boolean} quickOpen */
+  set quickOpen(quickOpen) {
+    this.menuSurface_.quickOpen = quickOpen;
   }
 
-  /** @return {number} */
-  get selectedItemIndex() {
-    return this.foundation_.getSelectedIndex();
+  /** @param {boolean} isFixed */
+  setFixedPosition(isFixed) {
+    this.menuSurface_.setFixedPosition(isFixed);
   }
 
-  /** @param {!boolean} rememberSelection */
-  set rememberSelection(rememberSelection) {
-    this.foundation_.setRememberSelection(rememberSelection);
+  hoistMenuToBody() {
+    this.menuSurface_.hoistMenuToBody();
   }
 
-  /** @param {boolean} quickOpen */
-  set quickOpen(quickOpen) {
-    this.foundation_.setQuickOpen(quickOpen);
+  /** @param {boolean} isHoisted */
+  setIsHoisted(isHoisted) {
+    this.menuSurface_.setIsHoisted(isHoisted);
+  }
+
+  /**
+   * @param {number} x
+   * @param {number} y
+   */
+  setAbsolutePosition(x, y) {
+    this.menuSurface_.setAbsolutePosition(x, y);
+  }
+
+  /**
+   * Sets the element that the menu-surface is anchored to.
+   * @param {!HTMLElement} element
+   */
+  setAnchorElement(element) {
+    this.menuSurface_.anchorElement = element;
+  }
+
+  registerListeners_() {
```

Comment:
I'm curious why we are registering and deregistering listeners based on when it's open/closed. Is there any harm to just having them hooked up all the time? (It'd be one thing if they were registered on the body, but these are on the menu surface element.)